[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x140fdfc]

Hi there, great app

trying to install controller on a debian machine and worker on a piB2

on reset getting

```
? Going to reset all of the hosts, which will destroy all configuration and data, Are you sure? Yes
INFO ==> Running phase: Connect to hosts
INFO [ssh] 10.0.22.166:22: connected
INFO [ssh] 10.0.22.127:22: connected
INFO ==> Running phase: Detect host operating systems
INFO [ssh] 10.0.22.166:22: overriding OS to debian
INFO [ssh] 10.0.22.127:22: overriding OS to debian
INFO [ssh] 10.0.22.166:22: is running Debian GNU/Linux 10 (buster)
INFO [ssh] 10.0.22.127:22: is running Raspbian GNU/Linux 10 (buster)
INFO ==> Running phase: Prepare hosts
INFO ==> Running phase: Gather k0s facts
INFO [ssh] 10.0.22.166:22: found existing configuration
INFO [ssh] 10.0.22.127:22: is running k0s worker version 0.13.1
INFO [ssh] 10.0.22.166:22: checking if worker  has joined
INFO ==> Running phase: Reset hosts
INFO [ssh] 10.0.22.166:22: running k0s reset
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x140fdfc]

goroutine 87 [running]:
github.com/k0sproject/rig/os.Linux.ServiceIsRunning(0x0, 0x0, 0x1982318, 0xc0005262c0, 0xc0003def70, 0x9, 0x9)
	github.com/k0sproject/rig@v0.3.17/os/linux.go:79 +0x5c
github.com/k0sproject/k0sctl/phase.(*Reset).Run.func1(0xc0005262c0, 0xc000555650, 0x0)
	github.com/k0sproject/k0sctl/phase/reset.go:50 +0xcd
github.com/k0sproject/k0sctl/config/cluster.(*Hosts).ParallelEach.func1(0xc000483920, 0xc000488178, 0xc0005262c0)
	github.com/k0sproject/k0sctl/config/cluster/hosts.go:87 +0xd8
created by github.com/k0sproject/k0sctl/config/cluster.(*Hosts).ParallelEach
	github.com/k0sproject/k0sctl/config/cluster/hosts.go:86 +0xe5
```

--- a/go.mod
+++ b/go.mod
@@ -15,7 +15,7 @@ require (
 	github.com/go-playground/validator/v10 v10.4.1
 	github.com/hashicorp/go-version v1.2.1
 	github.com/k0sproject/dig v0.1.1
-	github.com/k0sproject/rig v0.3.18
+	github.com/k0sproject/rig v0.3.19
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/mattn/go-isatty v0.0.12
 	github.com/segmentio/analytics-go v3.1.0+incompatible

